internal/bminventory/inventory.go

```diff
@@ -329,6 +329,9 @@ func (b *bareMetalInventory) setDefaultRegisterClusterParams(_ context.Context,
 	if params.NewClusterParams.Hyperthreading == nil {
 		params.NewClusterParams.Hyperthreading = swag.String(models.ClusterHyperthreadingAll)
 	}
+	if params.NewClusterParams.NetworkType == nil {
+		params.NewClusterParams.NetworkType = swag.String(models.ClusterNetworkTypeOVNKubernetes)
+	}
```

Comment:
Can we keep the current default behavior?  Leave NetworkType as optional, and if it's not set when we generate the install config then we'll choose as before?

Reply:
I don't think so, it will be conflicted between the non-rest call if no type will be selected we can't validate it.

Reply:
What do you mean by non-rest call?  We can say that the validation succeeds automatically if no type is selected.

Reply:
NM, added a commit that will set default type in case a type is not selected